Mohammed/index.js

```diff
@@ -0,0 +1,42 @@
+
+// 1. NaN
+
+// a-
+// NaN stands for 'Not A Number'
+
+// b-
+let checkForNaN = 'hello' * 1;
+console.log((checkForNaN !== NaN) ? 
```

Comment:
The result should be true or false and not NaN
like for example: console.log(isNaN("hello" *1));